src/main/java/com/amazon/dlic/auth/ldap2/LDAPAuthorizationBackend2.java

```diff
@@ -91,6 +92,7 @@ public LDAPAuthorizationBackend2(final Settings settings, final Path configPath)
         this.connectionFactory = ldapConnectionFactoryFactory.createConnectionFactory(this.connectionPool);
         this.userSearcher = new LDAPUserSearcher(settings);
         this.returnAttributes = settings.getAsList(ConfigConstants.LDAP_RETURN_ATTRIBUTES, Arrays.asList(ReturnAttributes.ALL.value())).toArray(new String[0]);
+        this.shouldFollowReferrals = settings.getAsBoolean(ConfigConstants.FOLLOW_REFERRALS, true);
```

Comment:
instead of `true` here as default we should use `ConfigConstants.FOLLOW_REFERRALS_DEFAULT`